Cython/Includes/libcpp/string.pxd

```diff
@@ -175,12 +180,37 @@ cdef extern from "<string>" namespace "std" nogil:
         bint operator>= (const char*)
 
 
-    string to_string(int val)
-    string to_string(long val)
-    string to_string (long long val)
-    string to_string (unsigned val)
-    string to_string (unsigned long val)
-    string to_string (unsigned long long val)
-    string to_string (float val)
-    string to_string (double val)
-    string to_string (long double val)
+    string to_string(int)
+    string to_string(long)
+    string to_string(long long)
+    string to_string(unsigned)
+    string to_string(unsigned long)
+    string to_string(unsigned long long)
+    string to_string(float)
+    string to_string(double)
+    string to_string(long double)
```

Comment:
1) I think it's nice to allow keyword arguments by naming the arguments – although, admittedly, it doesn't seem very attractive here, but there's still a slight chance that this breaks someone's code out there.
2) if we're changing these anyway, then we should also add `except +` since the allocation may fail.

Reply:
I removed the parameter names for consistency with the rest of the file. I did not add an exception clause, because none of the other functions that may throw an allocation exception have one. Only explicit exceptions such as out_of_range or so are caught here.

Reply:
BTW these `to_string` functions throw a compiler error about ambiguous signatures if you pass a `size_t` without an explicit cast to `unsigned long`. I am not quite sure why that is and it if is a special Cython quirk. To fix that I contemplated adding an explicit `size_t` overload, but ultimately decided against it, because it's not an official overload from the C++ documentation (although it would work). What's your stance on that?

Reply:
> I did not add an exception clause, because none of the other functions that may throw an allocation exception have one.

They can raise, so they'd better have one for safety.

> To fix that I contemplated adding an explicit `size_t` overload, but ultimately decided against it, because it's not an official overload from the C++ documentation (although it would work). What's your stance on that?

Sounds like a helpful addition. Also `ssize_t`. The declarations are meant to make Cython users happy, not the spec.

Reply:
Done.